components/brave_ads/core/internal/account/confirmations/payload/reward_confirmation_payload_util.cc

```diff
@@ -22,6 +22,8 @@ constexpr char kPublicKeyKey[] = "publicKey";
 
 base::Value::Dict BuildRewardConfirmationPayload(const RewardInfo& reward) {
   base::Value::List list;
+  list.reserve(1);
```

Comment:
no need to reserve place for 1 element: there is no optimization for such a case